internal/xcrd/schemas.go

```diff
@@ -29,6 +29,10 @@ const (
 // when translating an XRC into an XR.
 var PropagateClaimSpecProps = []string{"compositionRef", "compositionSelector", "compositionRevisionRef", "compositionUpdatePolicy"}
 
+// PropagateCompositeSpecProps is the list of XR spec properties to propagate
+// when translating an XR into an XRC.
+var PropagateCompositeSpecProps = []string{"compositionRef", "compositionSelector"}
```

Comment:
I'm pretty sure claims have `compositionRevisionRef` and `compositionUpdatePolicy` too (needs double checking). If so, this list would be identical to `PropagateClaimSpecProps` above. Maybe we could just call it something like `PropagateSpecProps` and use it in both places. 🤔

Reply:
> needs double checking

I only see `CompositeResourceDefinition` in the API Docs here - https://doc.crds.dev/github.com/crossplane/crossplane/apiextensions.crossplane.io/CompositeResourceDefinition/v1@v1.5.0

And the "Create CompositeResourceDefinition" example [here](https://crossplane.io/docs/v1.5/getting-started/create-configuration.html#create-compositeresourcedefinition) uses the same YAML to define the schema for the XRC and the XR so I believe you were right :) 

Will update to using a single `PropagateSpecProps` variable for both.

Reply:
Triple checked by running `kubectl explain postgresqlinstance.spec`:

```
KIND:     PostgreSQLInstance
VERSION:  database.example.org/v1alpha1

RESOURCE: spec <Object>

DESCRIPTION:
     <empty>

FIELDS:
   compositionRef	<Object>

   compositionRevisionRef	<Object>

   compositionSelector	<Object>

   compositionUpdatePolicy	<string>

   parameters	<Object> -required-

   resourceRef	<Object>

   writeConnectionSecretToRef	<Object>
```